accessioning/inventorying-digital-media-accession.md

```diff
@@ -0,0 +1,135 @@
+AU, L for NYPL internal institutional archives)  
+```M```  
+
+* The collection number from CMS  
+```1087```  
+
+* A dash  
+```-```  
+
+* A sequential number of the object padded to 4 digits  
+```0001```  
+
+    * When there are no media objects in the
+collection log the sequential number will be 0001. Objects already recorded in the log will be listed in
+the right hand column. The sequential number will be the next number
+that hasn't been allocated.  For example, the
+tenth object in manuscript collection ```1087``` would be labeled:
+```M1087-0010```.  
+
+The id numbers Digital Preservation uses are only for differentiating one disk from another. Sequence is only relevant for counting a number of disks. This method of id may be deprecated.
+{: .label .label-red }
+
+## 3. Label media.
+
+* For floppy and zip disks, print the number in ink on a new
+label and affixed it to the media. Labeling media with a P-Touch is best.
+
+* For Optical media (CDs, DVDs), print the number
+on the media's plastic hub using a using a medium point Sharpie pen. House CDs or DVDs in a sleeve or case if they are not housed. Rehouse CDs or DVDs that are sleeved/cased together into individual sleeves or cases, unless the case is designed for multiple CDs or DVDs.  
+
+* Make sure no label adhesive sticks to moving parts of the media. Many media types have shutters, gates, or spools that need to move freely to work. Covering a shutter or other moving part with a label can damage the disk and the drive used to read media.  
+
+* Check in with Digital Preservation staff to discuss labeling other media types.  
+
+|Digital Preservation staff can label commercial software with a simple collection name.
+```Crouch``` for Stanley Crouch papers.|
+
+![](/digitalmediaseparation/media/image3.jpg)  
+
+## 4. Create a media log record.  
+
+ Enter the following
+information in the media log page for the collection.
+
+* id
+
+    * Enter the identification number for the media.
+
+* media format
+
+    * Enter or select the appropriate media type (for example: 5.25 inch floppy disk,
+3.5 inch floppy disk, CD-R, DVD-R, zip disk, hard drive, flash drive).  
+    * See [Identifying Digital Media](../sitevisits/identifying-digital-media){:target="_blank"} for media types.   
+
+* floppy density
+
+    * Select the appropriate floppy disk density from the drop down if available. Leave blank for other media types.  
+
+* manufacturer
+
+    * Enter manufacturer of the media if known. 
+
+* serial number (hd)
+
+    * Enter the serial number if the media object is a hard disk drive. Leave blank for other media types.  
+
+| Label transcript is an optional field. If you want to transcript a label you can enter text as it appears on the label. You can enter ```[illegible]``` for words you can't decipher.|  
+
+An example of a completed record for separation:
+
+![](/digitalmediaseparation/media/image1.png)
+
+Deprecated
+{: .label .label-red }
```

Comment:
Is it possible for this label to be in line with the section 5 header or below it? Just so it's closer to what is being deprecated.